Challenge-4/Bot_2/src/helper.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  CONFIGURATION_ABI,
+  CONFIGURATOR_PROXY,
+  CONFIGURATOR_PROXY_POLYGON,
+} from "./constants";
+import { AlertsResponse, ethers, getAlerts } from "forta-agent";
+import { AlertType } from "./types";
+import { LRUCache } from "lru-cache";
+
+export default class Helper {
+  private provider: ethers.providers.Provider;
+  constructor(provider: ethers.providers.Provider) {
+    this.provider = provider;
+  }
+
+  public secondsPerYear = 31536000;
+
+  public async getProtocolConfiguration(
```

Comment:
Assuming that the return value of `getConfiguration()` doesn't change _every_ time it's called, you could probably implement caching to prevent calls where the return value will be the same.